docs/core/whats-new/dotnet-9/overview.md

```diff
@@ -17,19 +17,9 @@ New for .NET 9, the engineering team posts .NET 9 preview updates on [GitHub Dis
 
 The .NET 9 runtime includes a new attribute model for feature switches with trimming support. The new attributes make it possible to define [feature switches](https://github.com/dotnet/designs/blob/main/accepted/2020/feature-switch.md) that libraries can use to toggle areas of functionality.
 
-The runtime also includes numerous performance improvements in the following areas:
+Garbage collection includes a *dynamic adaptation to application size* feature.
```

Comment:
The feature is more impactful than the wording suggests. DATAS replaces "server GC" as the default serve GC. It's not a "when you need it, opt-in feature".

@mangod9 @Maoni0